{"created": 1758441872.682907, "duration": 0.09528231620788574, "exitcode": 1, "root": "/home/rts/\u4e0b\u8f7d/Python_src/mutants", "environment": {}, "summary": {"failed": 10, "total": 10, "collected": 10}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": "tests", "type": "Package"}]}, {"nodeid": "tests/test_add_values.py", "outcome": "passed", "result": [{"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput0]", "type": "Function", "lineno": 106}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput1]", "type": "Function", "lineno": 106}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput2]", "type": "Function", "lineno": 106}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput3]", "type": "Function", "lineno": 106}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput4]", "type": "Function", "lineno": 106}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput5]", "type": "Function", "lineno": 106}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput6]", "type": "Function", "lineno": 106}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput7]", "type": "Function", "lineno": 106}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput8]", "type": "Function", "lineno": 106}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput9]", "type": "Function", "lineno": 106}]}, {"nodeid": "tests", "outcome": "passed", "result": [{"nodeid": "tests/test_add_values.py", "type": "Module"}]}], "tests": [{"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput0]", "lineno": 106, "outcome": "failed", "keywords": ["test_add_values_with_func[originalInput0]", "parametrize", "pytestmark", "originalInput0", "test_add_values.py", "tests", "mutants", ""], "setup": {"duration": 0.0003196890029357746, "outcome": "passed"}, "call": {"duration": 0.00015831498603802174, "outcome": "failed", "crash": {"path": "/home/rts/\u4e0b\u8f7d/Python_src/mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nassert None is not None"}, "traceback": [{"path": "mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError"}], "longrepr": "originalInput = [1, 3, 2, 6, 9]\n\n    @pytest.mark.parametrize(\"originalInput\", [\n        [1, 3, 2, 6, 9],\n        [2, 1, 4, 4, 2],\n        [4, -2, 4, 6, 2],\n        [9, 2, 1, 5, 3, 2],\n        [-1, 9, 1, -3, -3],\n        [8, 3, 2, 6, 2, 3],\n        [1, 2, 3, 4, -5, 6],\n        [1, 2, 4, 2, 7, 5],\n        [1, 1, 2, 4, 1, 2],\n        [-2, 3, 1, 4, 7],\n    ])\n    \n    def test_add_values_with_func(originalInput):\n>       assert CURRENT_MUTANT_FUNC is not None, \"mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\"\nE       AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nE       assert None is not None\n\nmutants/tests/test_add_values.py:121: AssertionError"}, "teardown": {"duration": 0.00011538300896063447, "outcome": "passed"}}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput1]", "lineno": 106, "outcome": "failed", "keywords": ["test_add_values_with_func[originalInput1]", "parametrize", "pytestmark", "originalInput1", "test_add_values.py", "tests", "mutants", ""], "setup": {"duration": 0.00010775000555440784, "outcome": "passed"}, "call": {"duration": 0.00012341700494289398, "outcome": "failed", "crash": {"path": "/home/rts/\u4e0b\u8f7d/Python_src/mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nassert None is not None"}, "traceback": [{"path": "mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError"}], "longrepr": "originalInput = [2, 1, 4, 4, 2]\n\n    @pytest.mark.parametrize(\"originalInput\", [\n        [1, 3, 2, 6, 9],\n        [2, 1, 4, 4, 2],\n        [4, -2, 4, 6, 2],\n        [9, 2, 1, 5, 3, 2],\n        [-1, 9, 1, -3, -3],\n        [8, 3, 2, 6, 2, 3],\n        [1, 2, 3, 4, -5, 6],\n        [1, 2, 4, 2, 7, 5],\n        [1, 1, 2, 4, 1, 2],\n        [-2, 3, 1, 4, 7],\n    ])\n    \n    def test_add_values_with_func(originalInput):\n>       assert CURRENT_MUTANT_FUNC is not None, \"mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\"\nE       AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nE       assert None is not None\n\nmutants/tests/test_add_values.py:121: AssertionError"}, "teardown": {"duration": 9.845300519373268e-05, "outcome": "passed"}}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput2]", "lineno": 106, "outcome": "failed", "keywords": ["test_add_values_with_func[originalInput2]", "parametrize", "pytestmark", "originalInput2", "test_add_values.py", "tests", "mutants", ""], "setup": {"duration": 9.732600301504135e-05, "outcome": "passed"}, "call": {"duration": 0.00013872800627723336, "outcome": "failed", "crash": {"path": "/home/rts/\u4e0b\u8f7d/Python_src/mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nassert None is not None"}, "traceback": [{"path": "mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError"}], "longrepr": "originalInput = [4, -2, 4, 6, 2]\n\n    @pytest.mark.parametrize(\"originalInput\", [\n        [1, 3, 2, 6, 9],\n        [2, 1, 4, 4, 2],\n        [4, -2, 4, 6, 2],\n        [9, 2, 1, 5, 3, 2],\n        [-1, 9, 1, -3, -3],\n        [8, 3, 2, 6, 2, 3],\n        [1, 2, 3, 4, -5, 6],\n        [1, 2, 4, 2, 7, 5],\n        [1, 1, 2, 4, 1, 2],\n        [-2, 3, 1, 4, 7],\n    ])\n    \n    def test_add_values_with_func(originalInput):\n>       assert CURRENT_MUTANT_FUNC is not None, \"mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\"\nE       AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nE       assert None is not None\n\nmutants/tests/test_add_values.py:121: AssertionError"}, "teardown": {"duration": 0.00013768200005870312, "outcome": "passed"}}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput3]", "lineno": 106, "outcome": "failed", "keywords": ["test_add_values_with_func[originalInput3]", "parametrize", "pytestmark", "originalInput3", "test_add_values.py", "tests", "mutants", ""], "setup": {"duration": 0.0001084320101654157, "outcome": "passed"}, "call": {"duration": 0.00012158299796283245, "outcome": "failed", "crash": {"path": "/home/rts/\u4e0b\u8f7d/Python_src/mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nassert None is not None"}, "traceback": [{"path": "mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError"}], "longrepr": "originalInput = [9, 2, 1, 5, 3, 2]\n\n    @pytest.mark.parametrize(\"originalInput\", [\n        [1, 3, 2, 6, 9],\n        [2, 1, 4, 4, 2],\n        [4, -2, 4, 6, 2],\n        [9, 2, 1, 5, 3, 2],\n        [-1, 9, 1, -3, -3],\n        [8, 3, 2, 6, 2, 3],\n        [1, 2, 3, 4, -5, 6],\n        [1, 2, 4, 2, 7, 5],\n        [1, 1, 2, 4, 1, 2],\n        [-2, 3, 1, 4, 7],\n    ])\n    \n    def test_add_values_with_func(originalInput):\n>       assert CURRENT_MUTANT_FUNC is not None, \"mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\"\nE       AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nE       assert None is not None\n\nmutants/tests/test_add_values.py:121: AssertionError"}, "teardown": {"duration": 7.840000034775585e-05, "outcome": "passed"}}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput4]", "lineno": 106, "outcome": "failed", "keywords": ["test_add_values_with_func[originalInput4]", "parametrize", "pytestmark", "originalInput4", "test_add_values.py", "tests", "mutants", ""], "setup": {"duration": 9.296799544245005e-05, "outcome": "passed"}, "call": {"duration": 0.00010842199844773859, "outcome": "failed", "crash": {"path": "/home/rts/\u4e0b\u8f7d/Python_src/mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nassert None is not None"}, "traceback": [{"path": "mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError"}], "longrepr": "originalInput = [-1, 9, 1, -3, -3]\n\n    @pytest.mark.parametrize(\"originalInput\", [\n        [1, 3, 2, 6, 9],\n        [2, 1, 4, 4, 2],\n        [4, -2, 4, 6, 2],\n        [9, 2, 1, 5, 3, 2],\n        [-1, 9, 1, -3, -3],\n        [8, 3, 2, 6, 2, 3],\n        [1, 2, 3, 4, -5, 6],\n        [1, 2, 4, 2, 7, 5],\n        [1, 1, 2, 4, 1, 2],\n        [-2, 3, 1, 4, 7],\n    ])\n    \n    def test_add_values_with_func(originalInput):\n>       assert CURRENT_MUTANT_FUNC is not None, \"mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\"\nE       AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nE       assert None is not None\n\nmutants/tests/test_add_values.py:121: AssertionError"}, "teardown": {"duration": 7.388900849036872e-05, "outcome": "passed"}}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput5]", "lineno": 106, "outcome": "failed", "keywords": ["test_add_values_with_func[originalInput5]", "parametrize", "pytestmark", "originalInput5", "test_add_values.py", "tests", "mutants", ""], "setup": {"duration": 8.544400043319911e-05, "outcome": "passed"}, "call": {"duration": 0.00010445700900163502, "outcome": "failed", "crash": {"path": "/home/rts/\u4e0b\u8f7d/Python_src/mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nassert None is not None"}, "traceback": [{"path": "mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError"}], "longrepr": "originalInput = [8, 3, 2, 6, 2, 3]\n\n    @pytest.mark.parametrize(\"originalInput\", [\n        [1, 3, 2, 6, 9],\n        [2, 1, 4, 4, 2],\n        [4, -2, 4, 6, 2],\n        [9, 2, 1, 5, 3, 2],\n        [-1, 9, 1, -3, -3],\n        [8, 3, 2, 6, 2, 3],\n        [1, 2, 3, 4, -5, 6],\n        [1, 2, 4, 2, 7, 5],\n        [1, 1, 2, 4, 1, 2],\n        [-2, 3, 1, 4, 7],\n    ])\n    \n    def test_add_values_with_func(originalInput):\n>       assert CURRENT_MUTANT_FUNC is not None, \"mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\"\nE       AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nE       assert None is not None\n\nmutants/tests/test_add_values.py:121: AssertionError"}, "teardown": {"duration": 7.319600263144821e-05, "outcome": "passed"}}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput6]", "lineno": 106, "outcome": "failed", "keywords": ["test_add_values_with_func[originalInput6]", "parametrize", "pytestmark", "originalInput6", "test_add_values.py", "tests", "mutants", ""], "setup": {"duration": 8.928700117394328e-05, "outcome": "passed"}, "call": {"duration": 0.00010586499411147088, "outcome": "failed", "crash": {"path": "/home/rts/\u4e0b\u8f7d/Python_src/mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nassert None is not None"}, "traceback": [{"path": "mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError"}], "longrepr": "originalInput = [1, 2, 3, 4, -5, 6]\n\n    @pytest.mark.parametrize(\"originalInput\", [\n        [1, 3, 2, 6, 9],\n        [2, 1, 4, 4, 2],\n        [4, -2, 4, 6, 2],\n        [9, 2, 1, 5, 3, 2],\n        [-1, 9, 1, -3, -3],\n        [8, 3, 2, 6, 2, 3],\n        [1, 2, 3, 4, -5, 6],\n        [1, 2, 4, 2, 7, 5],\n        [1, 1, 2, 4, 1, 2],\n        [-2, 3, 1, 4, 7],\n    ])\n    \n    def test_add_values_with_func(originalInput):\n>       assert CURRENT_MUTANT_FUNC is not None, \"mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\"\nE       AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nE       assert None is not None\n\nmutants/tests/test_add_values.py:121: AssertionError"}, "teardown": {"duration": 7.066999387461692e-05, "outcome": "passed"}}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput7]", "lineno": 106, "outcome": "failed", "keywords": ["test_add_values_with_func[originalInput7]", "parametrize", "pytestmark", "originalInput7", "test_add_values.py", "tests", "mutants", ""], "setup": {"duration": 8.26010073069483e-05, "outcome": "passed"}, "call": {"duration": 0.00010194300557486713, "outcome": "failed", "crash": {"path": "/home/rts/\u4e0b\u8f7d/Python_src/mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nassert None is not None"}, "traceback": [{"path": "mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError"}], "longrepr": "originalInput = [1, 2, 4, 2, 7, 5]\n\n    @pytest.mark.parametrize(\"originalInput\", [\n        [1, 3, 2, 6, 9],\n        [2, 1, 4, 4, 2],\n        [4, -2, 4, 6, 2],\n        [9, 2, 1, 5, 3, 2],\n        [-1, 9, 1, -3, -3],\n        [8, 3, 2, 6, 2, 3],\n        [1, 2, 3, 4, -5, 6],\n        [1, 2, 4, 2, 7, 5],\n        [1, 1, 2, 4, 1, 2],\n        [-2, 3, 1, 4, 7],\n    ])\n    \n    def test_add_values_with_func(originalInput):\n>       assert CURRENT_MUTANT_FUNC is not None, \"mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\"\nE       AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nE       assert None is not None\n\nmutants/tests/test_add_values.py:121: AssertionError"}, "teardown": {"duration": 7.304499740712345e-05, "outcome": "passed"}}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput8]", "lineno": 106, "outcome": "failed", "keywords": ["test_add_values_with_func[originalInput8]", "parametrize", "pytestmark", "originalInput8", "test_add_values.py", "tests", "mutants", ""], "setup": {"duration": 8.87060014065355e-05, "outcome": "passed"}, "call": {"duration": 0.00010393401316832751, "outcome": "failed", "crash": {"path": "/home/rts/\u4e0b\u8f7d/Python_src/mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nassert None is not None"}, "traceback": [{"path": "mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError"}], "longrepr": "originalInput = [1, 1, 2, 4, 1, 2]\n\n    @pytest.mark.parametrize(\"originalInput\", [\n        [1, 3, 2, 6, 9],\n        [2, 1, 4, 4, 2],\n        [4, -2, 4, 6, 2],\n        [9, 2, 1, 5, 3, 2],\n        [-1, 9, 1, -3, -3],\n        [8, 3, 2, 6, 2, 3],\n        [1, 2, 3, 4, -5, 6],\n        [1, 2, 4, 2, 7, 5],\n        [1, 1, 2, 4, 1, 2],\n        [-2, 3, 1, 4, 7],\n    ])\n    \n    def test_add_values_with_func(originalInput):\n>       assert CURRENT_MUTANT_FUNC is not None, \"mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\"\nE       AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nE       assert None is not None\n\nmutants/tests/test_add_values.py:121: AssertionError"}, "teardown": {"duration": 9.278798825107515e-05, "outcome": "passed"}}, {"nodeid": "tests/test_add_values.py::test_add_values_with_func[originalInput9]", "lineno": 106, "outcome": "failed", "keywords": ["test_add_values_with_func[originalInput9]", "parametrize", "pytestmark", "originalInput9", "test_add_values.py", "tests", "mutants", ""], "setup": {"duration": 0.00015916299889795482, "outcome": "passed"}, "call": {"duration": 0.0001438979961676523, "outcome": "failed", "crash": {"path": "/home/rts/\u4e0b\u8f7d/Python_src/mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nassert None is not None"}, "traceback": [{"path": "mutants/tests/test_add_values.py", "lineno": 121, "message": "AssertionError"}], "longrepr": "originalInput = [-2, 3, 1, 4, 7]\n\n    @pytest.mark.parametrize(\"originalInput\", [\n        [1, 3, 2, 6, 9],\n        [2, 1, 4, 4, 2],\n        [4, -2, 4, 6, 2],\n        [9, 2, 1, 5, 3, 2],\n        [-1, 9, 1, -3, -3],\n        [8, 3, 2, 6, 2, 3],\n        [1, 2, 3, 4, -5, 6],\n        [1, 2, 4, 2, 7, 5],\n        [1, 1, 2, 4, 1, 2],\n        [-2, 3, 1, 4, 7],\n    ])\n    \n    def test_add_values_with_func(originalInput):\n>       assert CURRENT_MUTANT_FUNC is not None, \"mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\"\nE       AssertionError: mutant_func \u6ca1\u6709\u88ab\u6ce8\u5165\nE       assert None is not None\n\nmutants/tests/test_add_values.py:121: AssertionError"}, "teardown": {"duration": 7.618300151079893e-05, "outcome": "passed"}}], "warnings": [{"message": "Unknown config option: pytest -vv --tb\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/rts/anaconda3/envs/mutmut39/lib/python3.9/site-packages/_pytest/config/__init__.py", "lineno": 1474}]}